tests/namespace_test/namespace_test2_generated.ts

```diff
@@ -1,6 +1,6 @@
 // automatically generated by the FlatBuffers compiler, do not modify
 
-import * as NS4989953370203581498 from "./namespace_test1_generated";
```

Comment:
Hmm.. would be worth generating more stable identifiers then.. or leaving these changes out of the PR?